cypress/e2e/tests/setup/rancher-setup.spec.ts

```diff
@@ -8,7 +8,7 @@ describe('Rancher setup', { tags: ['@adminUserSetup', '@standardUserSetup', '@se
     cy.visit('');
 
     new RancherSetupPagePo().hasInfoMessage();
-    cy.url().should('equal', `${ Cypress.config().baseUrl }/auth/login`);
```

Comment:
This appeared to have a race-condition which was being exacerbated by these changes causing this test to be more flaky. Depending on when the url is checked it may contain the timed-out query param.

Reply:
I guess there are query paramters added or?

Reply:
Correct ?timed-out.

![image](https://github.com/rancher/dashboard/assets/55104481/c84faec2-b810-4b2e-a54d-a38f6fa932ce)